### ISSUE TITLE
Documentation on custom loss function

### ISSUE THREAD

**Issue by rustrust** [LaurentMazare/tch-rs#391]
I would like to have a custom loss function for my neural network. Would it be possible to have some documentation on how to do this? Thanks!

**Comment by rustrust** [comment]
After a bit experimentation with the vae and mnist examples, this is actually rather easy to do. Thanks for the great library!
